jbpm-bpmn2/src/test/resources/BPMN2-DataInputAssociations-Pojo-Complete.bpmn2

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?> 
+<definitions id="Definition"
+             targetNamespace=""
+             typeLanguage="http://www.java.com/javaTypes"
+             expressionLanguage="http://www.mvel.org/2.0"
+             xmlns="http://www.omg.org/spec/BPMN/20100524/MODEL"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://www.omg.org/spec/BPMN/20100524/MODEL BPMN20.xsd"
+             xmlns:g="http://www.jboss.org/drools/flow/gpd"
+             xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI"
+             xmlns:dc="http://www.omg.org/spec/DD/20100524/DC"
+             xmlns:di="http://www.omg.org/spec/DD/20100524/DI"
+             xmlns:tns="http://www.jboss.org/drools">
+
+  <itemDefinition id="_instanceMetadataItem" structureRef="org.jbpm.bpmn2.DataTest$Person" />
+
+  <process processType="Private" isExecutable="true" id="process" name="process" >
+  
+<!-- process variables -->
+    <property id="instanceMetadata" itemSubjectRef="_instanceMetadataItem"/>
+
+    <!-- nodes -->
+    <startEvent id="_1" name="" />
+    <userTask id="_2" name="Task">
+      <ioSpecification>
+        <dataInput id='coId' name='coId'/>
+        <inputSet>
+          <dataInputRefs>coId</dataInputRefs>
+        </inputSet>
+        <outputSet>
+        </outputSet>
+      </ioSpecification>
+      <dataInputAssociation>
+        <sourceRef>instanceMetadata</sourceRef>
+        <targetRef>coId</targetRef>
+        <assignment language="http://www.mvel.org/2.0">
```

Comment:
_language_ is not a valid attribute inside _**tAssignment**_ type:
_The 'language' attribute is not allowed._

Reply:
yes, thats surprising, since bpmn specification https://www.omg.org/spec/BPMN/2.0/PDF, in page 223, states:
The Assignment class is used to specify a simple mapping of data elements using a specified Expression language.The default Expression language for all Expressions is specified in the Definitions element, using the expressionLanguage attribute. It can also be overridden on each individual Assignment using the same attribute
Anyway, I remove language attribute from XMLs and, in order to achieve independence from JIRA https://issues.redhat.com/browse/JBPM-9471, try to deduce if the language used in the assigment in the expression is mvel based on the context of from and to, but I left the code that search for language attribute in Assignment (which will always be null right now) so it gets executed when both XSD is fixed and Designed sets this attribute.